### DISCUSSION TITLE
Продать машину игроку

### DISCUSSION THREAD

**Discussion by Kvadratik34** [octothorp-team/public-dobrograd#655] [Предложения]
Предлагаю уже наконец-то ввести систему продажи своего автомобиля другому игроку. На многих GTA серверах это реализовано, а здесь ещё нет... А вообще, хочет ли эту систему видеть сообщество сервера?

**Comment by Ggman77** [comment]
В этом нету смысла, ведь покупка происходит через f4 menu и этим ни кто не будет подьзоваться

**Reply by Kvadratik34** [reply]
Почему не будут пользоваться? В официальном магазине автомобиль покупается за более высокую цену, а игроки смогут купить этот же автомобиль, только за более низкую цену.

**Comment by Kyper1** [comment]
Очень не хватает такого. К примеру покрасил машину, подобрал диски и спойлер, подходящие только этой машине, а потом бац и захотел продать, но понимаешь, что вряд ли кто-то купит отдельно все детали или же они не подходят тому автомобилю, который ты собираешься купить, а такая функция избавит игроков от этой проблемы.

**Comment by LENFYS** [comment]
За!
